client/src/components/VideoForm.js

```diff
@@ -2,154 +2,166 @@ import styles from "./VideoForm.module.css";
+      // Set noError state to false. This will help indicating whether we render a video later.
+      setNoErrors(true);
+    } else {
+      // Errors are present, handle them appropriately
+      console.log("Errors are:", tempErrors);
+    }
+  };
+
+  return (
+    <div>
+      <form className={styles["video-form"]} onSubmit={handleSubmit}>
+        <h1>LoopTube</h1>
+        <h3>Fill your form</h3>
+
+        <div className={styles["input-group"]}>
+          {/* We need to group <label> and <input> in same line        */}
+          <label>Youtube Url</label>
+          <input
+            type="text"
+            onChange={(e) => setUrl(e.target.value)}
+            value={url}
+            placeholder="Enter your youtube url here..."
+          />
+        </div>
+        <div className={styles["time-input-group"]}>
+          <label>Start (hh:mm:ss)</label>
+          <input
+            type="number"
+            min="0" //So the value of input cannot be negative
+            step="1" //only accept integer
+            onChange={(e) => setStartHour(e.target.value)}
+            value={startHour}
+          />
+          <span>:</span>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setStartMinute(e.target.value)}
+            value={startMinute}
+          />
+          <span>:</span>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setStartSecond(e.target.value)}
+            value={startSecond}
+          />
+        </div>
+        <div className={styles["time-input-group"]}>
+          <label>End (hh:mm:ss)</label>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setEndHour(e.target.value)}
+            value={endHour}
+          />
+          <span>:</span>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setEndMinute(e.target.value)}
+            value={endMinute}
+          />
+          <span>:</span>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setEndSecond(e.target.value)}
+            value={endSecond}
+          />
+        </div>
+        <div className={styles["input-group"]}>
+          <label>Customize</label>
+          <input
+            type="number"
+            min="0"
+            step="1"
+            onChange={(e) => setCustomize(e.target.value)}
+            value={customize}
+          />
+        </div>
+
+        <div className={styles["button-container"]}>
+          <button type="submit">Submit</button>
+        </div>
+
+        {errors.length > 0 && (
+          <div>
+            {errors.map((error, index) => (
+              <p key={index}>{error}</p>
+            ))}
+          </div>
+        )}
+      </form>
+
+      {noErrors && (
```

Comment:
- this is not a good way to show <Iframe> inside VideoForm.js. VideoForm is components only handle form input, iframe is the video. This set up will make future development harder
**- Recommend:** try to set up redux toolkit, or pass state value from VideoForm.js to Home.js file (pass state value from child-parent)